hvac/api/auth/mfa.py

```diff
@@ -0,0 +1,172 @@
+        :rtype: dict
+        """
+        api_path = '/v1/auth/{mount_point}/mfa_config'.format(
+            mount_point=mount_point,
+        )
+        response = self._adapter.get(url=api_path)
+        return response.json()
+
+    def configure_duo_access(self, mount_point, host, integration_key, secret_key):
+        """Configure the access keys and host for Duo API connections.
+
+        To authenticate users with Duo, the backend needs to know what host to connect to and must authenticate with an
+        integration key and secret key. This endpoint is used to configure that information.
+
+        Supported methods:
+            POST: /auth/{mount_point}/duo/access. Produces: 204 (empty body)
+
+        :param mount_point: The "path" the method/backend was mounted on.
+        :type mount_point: str | unicode
+        :param host: Duo API host
+        :type host: str | unicode
+        :param integration_key: Duo integration key
+        :type integration_key: Duo secret key
+        :param secret_key: The "path" the method/backend was mounted on.
+        :type secret_key: str | unicode
+        :return: The response of the configure_duo_access request.
+        :rtype: requests.Response
+        """
+        params = {
+            'host': host,
+            'ikey': integration_key,
+            'skey': secret_key,
+        }
+        api_path = '/v1/auth/{mount_point}/duo/access'.format(
+            mount_point=mount_point,
+        )
+        return self._adapter.post(
+            url=api_path,
+            json=params,
+        )
+
+    def configure_duo_behavior(self, mount_point, push_info=None, user_agent=None, username_format='%s'):
+        """Configure Duo second factor behavior.
+
+        This endpoint allows you to configure how the original auth method username maps to the Duo username by
+        providing a template format string.
+
+        Supported methods:
+            POST: /auth/{mount_point}/duo/config. Produces: 204 (empty body)
+
+
+        :param mount_point: The "path" the method/backend was mounted on.
+        :type mount_point: str | unicode
+        :param push_info: A string of URL-encoded key/value pairs that provides additional context about the
+            authentication attempt in the Duo Mobile app
+        :type push_info: str | unicode
+        :param user_agent: User agent to connect to Duo (default "")
+        :type user_agent: str | unicode
+        :param username_format: Format string given auth method username as argument to create Duo username
+            (default '%s')
+        :type username_format: str | unicode
+        :return: The response of the configure_duo_behavior request.
+        :rtype: requests.Response
+        """
+        params = {
+            'username_format': username_format,
+        }
+        if push_info is not None:
+            params['push_info'] = push_info
+        if user_agent is not None:
+            params['user_agent'] = user_agent
```

Comment:
"user_agent" is passed as a param, rather than a "User-Agent" header?  I guess the vault code [somewhere around hereabouts](https://github.com/hashicorp/vault/blob/8142b42d951119a73ce46daa3331921b5e21cdee/helper/mfa/duo/path_duo_config.go#L16-L19) translates it to a header when making a call to duo?

also, might it be reasonable to default to 'github.com/hvac' or similar?